src/pages/StableSwap/index.tsx

```diff
@@ -231,12 +252,15 @@ export default function StableSwap() {
           txHash: undefined
         })
       })
-  }, [tradeToConfirm, showConfirm, swapCallback])
+  }, [tradeToConfirm, showConfirm, swapCallback, priceImpactWithoutFee])
 
   // warnings on slippage
-  // @nocommit Add This
-  // const priceImpactSeverity = warningSeverity(priceImpactWithoutFee)
-  const priceImpactSeverity = 0
+  const priceImpactSeverity = isStableSwapHighPriceImpact(priceImpact)
```

Comment:
This might be useful:

```suggestion
  const priceImpactSeverity = isStableSwapHighPriceImpact(priceImpact)
  const showPriceImpactError = (priceImpactSeverity && !isExportMode)
```

Reply:
lgtm